Source/WebCore/css/CSSStyleRule.h

```diff
@@ -46,6 +47,10 @@ class CSSStyleRule final : public CSSRule, public CanMakeWeakPtr<CSSStyleRule> {
     // FIXME: Not CSSOM. Remove.
     StyleRule& styleRule() const { return m_styleRule.get(); }
 
+    CSSRuleList& cssRules() const;
```

Comment:
https://drafts.csswg.org/css-nesting/#issue-e535b5ff

This is a high probability that the spec will move this to CSSStyleDeclaration, but we can start with this until the spec is finalised.

Reply:
I purposely didn’t web-expose this since it’s still in flux. Just needed it somewhere to start, and for now matching the spec and CSSGroupingRule seemed like a safe choice.